File: Raven.Client.Java/src/test/java/net/ravendb/tests/bugs/transformresults/AnswerVote.java
package net.ravendb.tests.bugs.transformresults;

public class AnswerVote {
  private String questionId;
  private String answerId;
  private int delta;
  private double decimalValue;
  public String getQuestionId() {
    return questionId;
  }
  public void setQuestionId(String questionId) {
    this.questionId = questionId;
  }
  public String getAnswerId() {
    return answerId;
  }
  public void setAnswerId(String answerId) {
    this.answerId = answerId;
  }
  public int getDelta() {
    return delta;
  }
  public void setDelta(int delta) {
    this.delta = delta;
  }
  public double getDecimalValue() {
    return decimalValue;
  }
  public void setDecimalValue(double decimalValue) {
    this.decimalValue = decimalValue;
  }

}

File: test/SlowTests/Tests/Bugs/TransformResults/AnswerVote.cs
namespace SlowTests.Tests.Bugs.TransformResults
{
    public class AnswerVote
    {
        public string QuestionId { get; set; }
        public string AnswerId { get; set; }
        public int Delta { get; set; }
        public decimal DecimalValue { get; set; }
    }

    public class AnswerVote2
    {
        public string Id { get; set; }
        public string QuestionId { get; set; }
        public string AnswerId { get; set; }
        public int Delta { get; set; }
    }
}
